play-ebean/src/main/java/play/db/ebean/EbeanDynamicEvolutions.java

```diff
@@ -65,9 +66,22 @@ public void create() {
                 String evolutionScript = generateEvolutionScript(servers.get(key), serverConfig);
                 if (evolutionScript != null) {
                     File evolutions = environment.getFile("conf/evolutions/" + key + "/1.sql");
-                    if (!evolutions.exists() || Files.readFile(evolutions).startsWith("# --- Created by Ebean DDL")) {
-                        Files.createDirectory(environment.getFile("conf/evolutions/" + key));
-                        Files.writeFileIfChanged(evolutions, evolutionScript);
+                    try {
+                        String content;
+                        if (!evolutions.exists()) {
+                            content = "";
+                        } else {
+                            content = new String(Files.readAllBytes(evolutions.toPath()), "utf-8");
+                        }
+
+                        if (content.isEmpty() || content.startsWith("# --- Created by Ebean DDL")) {
+                            environment.getFile("conf/evolutions/" + key).mkdirs();
+                            if (!content.equals(evolutionScript)) {
+                                Files.write(evolutions.toPath(), evolutionScript.getBytes("utf-8"));
+                            }
+                        }
+                    } catch (IOException e) {
+                        throw new RuntimeException(e);
```

Comment:
James, what's the rationale for when to throw an exception like `RuntimeException` vs `UsefulException`.